ui.frontend/src/main/webpack/site/js/components/buildFacet.ts

```diff
@@ -49,7 +50,7 @@ BuildFacetOption): HTMLDivElement => {
     : false
 
   // eslint-disable-next-line @typescript-eslint/no-misused-promises
-  facetInput.addEventListener('change', async (event) => {
+  facetInput.addEventListener('SAAS-facets-select', async (event) => {
```

Comment:
As discussed, we should not change this event.

Reply:
This is done